recipes/python/retail/Dockerfile

```diff
@@ -30,6 +30,6 @@ COPY . .
 
 RUN python setup.py clean --all install
 
-RUN cp dist/retail*.egg /application.egg
+COPY dist/retail*.egg /application.egg
```

Comment:
I think we agreed to use "RUN cp" as we do not copy the egg file from local machine any longer. @shivmpatel12, can you please confirm or deny?

Reply:
Yup confirmed. `RUN cp dist/retail*.egg /application.egg` because the egg is generated on the container

Reply:
It did not work with RUN cp whenever I try to generate the docker image locally.

Reply:
This seems to happen when the egg file is generated locally. Reverting the changes.